src/app/components/success.tsx

```diff
@@ -0,0 +1,12 @@
+import style from '/styles/success.module.css'
+import Link from "../../../node_modules/next/link";
+
+export default function SuccessComponent() {
+    return (
+    <div className={style.box}>
+        <h1 className={style.title}>Success!</h1>
+        <h3 className={style.message}>You have successfully signed up for an account.</h3>
+        <Link href="/pages/login"><button className={style.login}>Back to Login</button></Link>
```

Comment:
We are using tailwindcss for styling components, for a quick summary: tailwindcss allows us to style tags using specific keywords in the className property of a tag. 

Ex- `<h1 className="text-center font-sans"> Success!</h1>`